package.json

```diff
@@ -19,8 +19,5 @@
   "homepage": "https://github.com/estefrare/BaSD-MindSet#readme",
   "dependencies": {
     "express": "^4.17.1"
```

Comment:
why did you remove `nodemon`?

Reply:
I thought that was what you pointed me

Reply:
I will leave the one in the development branch